files/image_config/get_graph/get_graph

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+while [ ! -f /etc/sonic/graph_service_url ]; do
+    echo "No graph service url speficied, waiting for DHCP response..."
```

Comment:
Yes, syslog.

---
In reply to: [100567923](https://github.com/Azure/sonic-buildimage/pull/288#discussion_r100567923) [](ancestors = 100567923)